source/_layouts/javapage.html

```diff
@@ -70,6 +72,80 @@ <h2 class="heading-blc">Java</h2>
                 <div class="box-container">
                   <div class="top-section-grp">
                     <div class="heading-block">
+                      <h2>Zuul Spring Cloud Starter</h2>
+                    </div>
+                    <div class="two-boxes-blc">
+                      <div class="row">
+                        <a href="/java/spring-cloud-zuul//quickstart.html">
+                          <div class="col-md-6 col-xs-12">
+                            <div class="single-bx">
+                              <div class="icon"><i class="fa fa-forward" aria-hidden="true"></i></div>
+                              <div class="text">Quickstart</div>
+                            </div>
+                          </div>
+                        </a>
+                        <a href="/java/spring-cloud-zuul//">
+                          <div class="col-md-6 col-xs-12">
+                            <div class="single-bx">
+                              <div class="icon"><i class="fa fa-book" aria-hidden="true"></i></div>
+                              <div class="text">Documentation</div>
+                            </div>
+                          </div>
+                        </a>
+                      </div>
+                    </div>
+                    <ul class="list-item">
+                      <li><a href="https://github.com/stormpath/stormpath-sdk-java/tree/master/examples/zuul-spring-cloud-starter"> <div class="icon"><i class="fa fa-angle-left" aria-hidden="true"></i> <i class="fa fa-angle-right" aria-hidden="true"></i></div>Example Application</a></li>
+                      <li><a href="https://github.com/stormpath/stormpath-sdk-java/tree/master/extensions/spring/cloud"><div class="icon"> <i class="fa fa-globe" aria-hidden="true"></i> </div>Project on GitHub</a></li>
+                    </ul>
+                  </div>
+                  <div class="text-container-blc">
+                    <p>For Spring Cloud Zuul Gateways.</p>
+                  </div>
+                  <div class="clearfix"></div>
+                </div>
+              </div>
+              <div class="box-container">
+                  <div class="top-section-grp">
+                    <div class="heading-block">
+                      <h2>Spring WebMVC</h2>
+                    </div>
+                    <div class="two-boxes-blc">
+                      <div class="row">
+                        <a href="/java/spring/quickstart.html">
+                          <div class="col-md-6 col-xs-12">
+                            <div class="single-bx">
+                              <div class="icon"><i class="fa fa-forward" aria-hidden="true"></i></div>
+                              <div class="text">Quickstart</div>
+                            </div>
+                          </div>
+                        </a>
+                        <a href="/java/spring/">
```

Comment:
please update to `/java/spring-web/`. this keeps is consistent with spring boot where we have `spring-boot-web`